Sources/SafeDICore/Extensions/ImportDeclSyntaxExtensions.swift

```diff
@@ -23,14 +23,11 @@ import SwiftSyntax
 extension ImportDeclSyntax {
     // MARK: Public
 
-    public var asImportStatement: ImportStatement? {
-        guard let moduleName = path.first?.name.text else {
```

Comment:
we really don't need to `guard` here – if our code-gen is running then our build succeeded, which means we don't have any dangling `import` statements.